### PR TITLE
Fix protobuf version and adjust docs

### PR DESCRIPTION
**Pull request by steffenkleinle**
### Short description

I had to upgrade the protobuf version to get it to work on my machine. Let me know if that is alright or whether I should revert it.

### Proposed changes

- Upgrade protobuf
- Adjust docs

### Side effects

None.

### Resolved issues

Fixes: None.

### CODE REVIEW

File: backend/build.gradle.kts [hunk 1]
```diff
@@ -24,7 +24,7 @@ repositories {
 }
 
 dependencies {
-    implementation("com.google.protobuf:protobuf-kotlin:3.25.2")
+    implementation("com.google.protobuf:protobuf-kotlin:4.27.2")
```
Comment (seluianova) [inline backend/build.gradle.kts:27]: Remembering how I struggled with the protobuff setup, I'm scared to test this change 😄

**Review by f1sh1918 — APPROVED** [review]
Tested:
- Running all protobuf creation scripts
- Created a card
- Activated a card

Looks like it works fine. Maybe someone else should doublecheck but i couldn't see any other breaking changes that may cause issues

**Review by seluianova — APPROVED** [review]
Seems to work for me

File: backend/build.gradle.kts (hunk 1 above)
Comment (steffenkleinle) [reply backend/build.gradle.kts:27]: @seluianova I get that, would be super cool if you could still test this though please :pray:

Comment (seluianova) [reply backend/build.gradle.kts:27]: maybe would be helpful to specify the exact version of proto compiler in the dev instruction?

**Pull request merged**